.eslintrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  env: {
+    es6: true,
+    node: true
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:import/recommended',
+    'plugin:mocha/recommended',
+    'plugin:node/recommended',
+    'plugin:promise/recommended',
+    'prettier-standard'
```

Comment:
I'm glad this exists, because otherwise getting these to play nicely seems like a PITA.